documentation/modules/deploying/con-deploy-prereqs.adoc

```diff
@@ -9,24 +9,38 @@ To deploy Strimzi, make sure:
 
 ifdef::Downloading[]
 * A Kubernetes {KubernetesVersion} cluster is available
-endif::Downloading[]
-ifndef::Downloading[]
-* An OpenShift {OpenShiftVersion} cluster is available
 +
-Strimzi is based on Strimzi {StrimziVersion}.
-endif::Downloading[]
+If you do not have access to a Kubernetes cluster, you can install Strimzi with xref:deploy-kubernetes-{context}[_Minikube_].
 * The `kubectl` command-line tool is installed and configured to connect to the running cluster.
 
 NOTE: Strimzi supports some features that are specific to OpenShift,
 where such integration benefits OpenShift users and there is no equivalent implementation using standard Kubernetes.
 
-ifdef::InstallationAppendix[]
 [discrete]
-== Alternatives if a Kubernetes cluster is not available
+== `oc` and `kubectl` commands
+
+The `oc` command functions as an alternative to `kubectl`.
+In almost all cases the example `kubectl` commands given in this guide can be done using `oc` simply by replacing the command name (options and arguments remain the same).
```

Comment:
_Suggested change:_

In almost all cases the example `kubectl` commands used in this guide can be done using `oc` simply by replacing the command name (options and arguments remain the same).